src/test/java/nl/atos/ontwikkelstraat/OntwikkelstraatApplicationTest.java

```diff
@@ -0,0 +1,31 @@
+package nl.atos.ontwikkelstraat;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Created by nerom on 31-3-2016.
+ */
+public class OntwikkelstraatApplicationTest {
+
+    OntwikkelstraatApplication application;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/vinsterdamus/Ontwikkelstraat/file/2347084577/issues/source?bid=3138095&fileBranchId=3208141#l14)